application/common/models/PasswordHistory.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace common\models;
+
+use common\helpers\Utils;
+use yii\helpers\ArrayHelper;
+
+class PasswordHistory extends PasswordHistoryBase
+{
+    public $password;
+
+    public function rules(): array
+    {
+        return ArrayHelper::merge([
+            [
+                'created_utc', 'default', 'value' => Utils::now(),
```

Comment:
Since `default` only applies if the the existing value is empty, should we instead do something that will force `created_utc` to be the current datetime?